test/server/cards/02_SHD/events/Pillage.spec.ts

```diff
@@ -0,0 +1,92 @@
+import { Location } from '../../../../../server/game/core/Constants';
+
+describe('Pillage', function() {
+    integration(function(contextRef) {
+        describe('Pillage\'s ability', function() {
+            it('should let the player target the opponent, and let the opponent discard 2 cards from their hand', function() {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['pillage'],
+                    },
+                    player2: {
+                        hand: ['alliance-xwing', 'battlefield-marine', 'imperial-interceptor', 'wampa']
+                    }
+                });
+
+                const { context } = contextRef;
+                const { player1, player2, pillage, allianceXwing, battlefieldMarine, imperialInterceptor, wampa } = context;
+
+                player1.clickCard(pillage);
+                player1.clickPrompt('Opponent');
+                expect(player2).toBeAbleToSelectExactly([
+                    allianceXwing, battlefieldMarine, imperialInterceptor, wampa
+                ]);
+
+                player2.clickCard(imperialInterceptor);
+                player2.clickCard(wampa);
```

Comment:
Since we're having some card selection bugs, do you mind adding a third clicked card (with `clickCardNonChecking()` to avoid getting an error) to ensure that everything still works right